modules/fedimint-mint-client/src/event.rs

```diff
@@ -0,0 +1,26 @@
+use fedimint_client::db::event_log::{Event, EventKind};
+use fedimint_core::core::ModuleKind;
+use fedimint_mint_common::{Nonce, KIND};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Serialize, Deserialize)]
+pub struct NoteCreated {
+    pub nonce: Nonce,
+}
```

Comment:
Feels like we haven't put enough thought into what data we'd want in such an event. If it's supposed to eventually drive state machines we'd need a lot more info. I think this event will become redundant/will need to be phased out.

Reply:
We can add fields as we go. Since notes are created e.g. during recovery, such fields will often will need to be `Option` anyway.


But yeah, we can always create new events and phase out the old ones.